### ISSUE TITLE
Add accent-color support

### ISSUE THREAD

**Issue by bryancasler** [4site-interactive-studios/engrid#169]
https://caniuse.com/?search=accent-color

**Comment by MichaelT372** [comment]
Not recommending to add because I'm not sure of the benefit (in fact, I can think of more negatives than benefits) and it has poor browser support.

**Comment by bryancasler** [comment]
I agree, going to close this out. This was more an idea when it first came out as a catchall for when EN was adding field markup in a different pattern that our ENgrid didn't yet have coverage for.
